lib/CMakeLists.txt

```diff
@@ -195,20 +239,15 @@ target_link_libraries(arango PUBLIC
   boost_boost
   arango_futures
   arango_static_strings
-  arango_crashhandler
```

Comment:
Why part of them was moved to new library, but another part just copied? Is it intentional? And if yes why?